ui/app/components/policy-form.hbs

```diff
@@ -55,35 +54,50 @@
         {{! EDITING - no file upload toggle}}
         <JsonEditor
           @title="Policy"
-          @helpText="You can use Alt+Tab (Option+Tab on MacOS) in the code editor to skip to the next field"
           @value={{@model.policy}}
           @valueUpdated={{action (mut @model.policy)}}
           @mode="ruby"
           @extraKeys={{hash Shift-Enter=(perform this.save)}}
           data-test-policy-editor
         />
       {{/if}}
+      <p class="is-size-9 has-text-grey has-bottom-margin-l has-top-margin-xs">
+        You can use Alt+Tab (Option+Tab on MacOS) in the code editor to skip to the next field.
+        {{! Only renders button (and modal) if not already in the "create policy" modal }}
+        {{#if @renderPolicyExampleModal}}
+          See
+          <button
+            type="button"
+            class="text-button has-text-info"
+            {{on "click" (fn (mut this.showTemplateModal))}}
+            data-test-policy-example
+          >
+            example template
+          </button>.
+        {{/if}}
```

Comment:
You could possibly add an `else` instead of using an `unless` on line 81. If you add an `else`, you may need to change some extra styles/helper classes so that it fits the designs.